app/policies/application_policy.rb

```diff
@@ -0,0 +1,55 @@
+class ApplicationPolicy
```

Comment:
I'll skim over the policies - as I think that's one of the key areas that you will be working on in the next step for this feature.

Reply:
Yep, at the moment the only ones that really do anything are the admin/agent ones, which behave roughly the same as they do currently, e.g. only admins can create users.  Nothing else really does anything